docker/app/conf/apache/vhosts.conf

```diff
@@ -0,0 +1,23 @@
+DirectoryIndex index.php index.html
+
+<VirtualHost *:80>
+  ServerName drupal-bootstrap.org
+  ServerAlias localhost bootstrap.kindlehl.tag1.io
```

Comment:
The server name can be that as it matches the canonical name of this site, but lets remove the tag1.io alias so as to make this generic. If this is the only and default vhost it doesn't matter. (and it should be, to ensure we can use this for a local dev stack as well.